### DISCUSSION TITLE
Is it possible to create a pterodactyl panel on github to create a web host?

### DISCUSSION THREAD

**Discussion by Vik362352** [community/community#147476] [Programming Help]
### Select Topic Area

Question

### Body

Is it possible to create a pterodactyl panel on github to create a web host?
It's just that I personally don't feel comfortable using nginx using the console

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @Vik362352!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
